src/components/DocsSwitcher/docsPicker.module.css

```diff
@@ -0,0 +1,80 @@
+.container {
```

Comment:
All variables should come from the global stylesheet. This can be accessed locally at:

`chainlink docs/node_modules/@chainlink/design-system/global-styles.css`

Reply:
including color and spacing if possible

Reply:
I updated the design and it should be pixel perfect now